doc/template.html

```diff
@@ -67,5 +67,36 @@ <h3 class="date">$date$</h3>
 $for(include-after)$
 $include-after$
 $endfor$
+<script type="text/javascript">
+  (function() {
+    // write out a basic 'dropdown_link' reference element
+    var dropdown_link = document.createElement('a');
```

Comment:
`button` elements add additional CSS. Using a `a` tag accomplishes what we want with minimal CSS additions.